src/main/flight/imu.c

```diff
@@ -87,14 +86,18 @@ static float throttleAngleScale;
 static float fc_acc;
 static float smallAngleCosZ = 0;
 
-static float magneticDeclination = 0.0f;       // calculated at startup from config
```

Comment:
me neither,
but if it is used in iNav then it should be maintained there ...
don't you think.